### ISSUE TITLE
calendar icon not showing up on right sidebar

### ISSUE THREAD

**Issue by doreszr** [liamcain/obsidian-calendar-plugin#262]
> **Before Submitting:** Double-check that you are running the latest version of the plugin. The bug might have already been fixed 😄

### Describe the bug

After installing the calendar plugin (and enabling it), the icon doesn't show up on the right sidebar and I can only access it on the left side.

### Steps to reproduce

1. Turned restricted mode off
2. Installed the Calendar plugin
3. Enabled the plugin
4. The Calendar icon didn't show up on the right sidebar

### Environment (please specify)

Windows

#### Obsidian Version (e.g. v0.10.6)

v0.15.9

**Comment by prdlik007** [comment]
The same experience. Obsidian 0.16.2. - Minimal Theme

**Comment by birv2** [comment]
Same experience. Default theme, Windows 10, Obs 1.0.3. Actually won't show up in any pane.

**Comment by gmfawcett** [comment]
Same as @birv2. The open-calendar option does not appear in the Command Palette, either!

**Comment by birv2** [comment]
I did find a fix finally. Someone in this issue list said to click on the divider between the central pane and the right pane, which made the calendar icon appear above the right pane. It did! Clicking on that icon opened the calendar. Happy to report that, and I hope that it works for others. Though it's certainly a strange way to open a plugin.

**Comment by NaturallyAsh** [comment]
Same problem for me. No icon in sidebar. Obsidian latest version, Minimal theme, Mac Pro M1.

**Comment by dougedmunds** [comment]
https://github.com/liamcain/obsidian-calendar-plugin/issues/262#issuecomment-1315378365

Needs a  "more" indicator like ">>"  when top-right icons are not visible because the vertical separator bar is too far to the right, to force vertical bar to move left, showing hidden icons at top.

**Comment by bhattpriyank3** [comment]
> 

oh my god ...this is crazy..i had been mad because of this issue. Thanks bro..i had uninstalled every plugin ..but still didn't got success..thank you so much

**Comment by imadjamil** [comment]
> [#262 (comment)](https://github.com/liamcain/obsidian-calendar-plugin/issues/262#issuecomment-1315378365)
> 
> Needs a "more" indicator like ">>" when top-right icons are not visible because the vertical separator bar is too far to the right, to force vertical bar to move left, showing hidden icons at top.

Holly yes, this was really well hidden!

**Comment by ricir** [comment]
> [#262 (comment)](https://github.com/liamcain/obsidian-calendar-plugin/issues/262#issuecomment-1315378365)
> 
> Needs a "more" indicator like ">>" when top-right icons are not visible because the vertical separator bar is too far to the right, to force vertical bar to move left, showing hidden icons at top.

Agree. I clicked on the separator and nothing happened, but when enlarging the right pane the Calendar Icon showed up and also another icon (Quick Share) which I actually had never seen.
